README.md

```diff
@@ -19,6 +19,21 @@ If this works, then you will need to look for where that "GITHUB_PAT" variable i
 
 ## Using this package in a DfE data dashboard
 
+### Adding analytics to your dashboard
+
+For analytics to function on your dashboard, you will need to:
+
+- request a Google Analytics key from the [Statistics Development Team](mailto:statistics.development@education.gov.uk)
+- create a html file with the javascript required for your dashboard to connect to Google Analytics
```

Comment:
Is there a missing step here about adding a line in ui.R file?

Reply:
Yes, good point! Added now.